test/unit/accuracy.test.js

```diff
@@ -77,15 +101,44 @@ const resultsStateClassification = {
     temp: 'categorical',
     play: 'categorical'
   },
-  accuracyCheckLabels: ['no', 'yes', 'yes', 'yes', 'yes', 'no']
+  accuracyCheckLabels: [1, 0, 0, 0, 0, 1],
+  accuracyCheckExamples: [[0,2], [1,2], [1,1], [1,0], [2,1], [2,0]],
+  featureNumberKey: {
+    'temp': {
+      'cool' : 0,
+      'mild' : 1,
+      'hot' : 2,
+    },
+    'play': {
+      'yes' : 0,
+      'no' : 1
+    },
+    'weather': {
+      'sunny' : 0,
+      'overcast' : 1,
+      'rainy': 2
+    }
```

Comment:
This set up feels particularly fragile because the `accuracyCheckLabels` and `accuracyCheckExamples` have to match up with the values in the `featureNumberKey` so the test data literally reads like 1s and 0s which is hard to puzzle through unless you know the system really well. Anyone have ideas for improvements?

Reply:
I think I noticed something similar while working on [this](https://github.com/code-dot-org/ml-playground/pull/233/files#diff-7f797aa32260ac4cbc8b43b04d62976c18e503bea6cc86be7448f935896ec3cfR1000-R1021) code.  It's tricky when state involves parallel arrays that aren't explicitly tied to each other.  I think one solution is in a simpler representation in the core state.  Would it be possible to combine these parallel arrays into an array of objects?

Reply:
Ok, I think I'm going to get these tests in and consider a refactor separately.